.circleci/config.yml

```diff
@@ -22,7 +22,7 @@ common: &common
     # https://support.circleci.com/hc/en-us/articles/115015426888-Clear-project-cache
     - restore_cache:
         keys:
-          - v3-deps-{{ .Environment.CIRCLE_JOB }}-{{ checksum "setup.py" }}-{{ checksum "tox.ini" }}-{{ checksum "requirements.txt" }}-{{ checksum "requirements_dev.txt" }}
+          - v4-deps-{{ .Environment.CIRCLE_JOB }}-{{ checksum "setup.py" }}-{{ checksum "tox.ini" }}-{{ checksum "requirements.txt" }}-{{ checksum "requirements_dev.txt" }}

```

Comment:
I had to do this because otherwise, tests were failing because the build was reusing an old Python environment where the example plugin was still referencing `BaseCrawler`. (I'm not entirely sure how this was happening behind the scenes, but it was, and this fixed it!) 😅